### ISSUE TITLE
Exception during game start on Skeld map

### ISSUE THREAD

**Issue by Ashesh3** [SkeldJS/SkeldJS#28]
First of all, thanks for this wonderful library, I am so surprised to see such a beautifully documented, well maintained piece of software.
Everything works great and flawlessly.

I seem to be getting this exception when a Skeld map starts:

Code:

```

    const client = new skeldjs.SkeldjsClient('2021.11.9', {
        chatMode: skeldjs.QuickChatMode.FreeChat
    })

    console.log("Connecting..");
    await client.connect(skeldjs.OfficialServers.AS, "testname");

    console.log("Joining game..");
    await client.joinGame(code);
    console.log("Joined game.");

    await client.myPlayer.control.checkName("testname");
    await client.myPlayer.control.checkColor(2);

    const pathfinder = new SkeldjsPathfinder.SkeldjsPathfinder(client);
    let players = Array.from(client.players.entries())
    pathfinder.follow(players[Math.floor(Math.random() * players.length)][1])
    console.log("Joined!");

```

```
node:internal/errors:464
    ErrorCaptureStackTrace(err);
    ^

RangeError [ERR_OUT_OF_RANGE]: The value of "offset" is out of range. It must be >= 0 and <= 12. Received 13
    at new NodeError (node:internal/errors:371:5)
    at boundsError (node:internal/buffer:86:9)
    at Buffer.readUInt8 (node:internal/buffer:252:5)
    at HazelReader.uint8 (C:\Users\Ashesh\Desktop\Projects\among-us\node_modules\@skeldjs\util\dist\lib\HazelReader.js:33:34)
    at HazelReader.bool (C:\Users\Ashesh\Desktop\Projects\among-us\node_modules\@skeldjs\util\dist\lib\HazelReader.js:64:21)
    at AutoDoorsSystem.Deserialize (C:\Users\Ashesh\Desktop\Projects\among-us\node_modules\@skeldjs\core\dist\lib\systems\AutoDoorsSystem.js:24:37)       
    at SkeldShipStatus.Deserialize (C:\Users\Ashesh\Desktop\Projects\among-us\node_modules\@skeldjs\core\dist\lib\objects\InnerShipStatus.js:43:24)       
    at new Networkable (C:\Users\Ashesh\Desktop\Projects\among-us\node_modules\@skeldjs\core\dist\lib\Networkable.js:32:22)
    at new InnerShipStatus (C:\Users\Ashesh\Desktop\Projects\among-us\node_modules\@skeldjs\core\dist\lib\objects\InnerShipStatus.js:20:9)
    at new SkeldShipStatus (C:\Users\Ashesh\Desktop\Projects\among-us\node_modules\@skeldjs\core\dist\lib\objects\SkeldShipStatus.js:16:9) {
  code: 'ERR_OUT_OF_RANGE'
}
```

Weirdly, this does not happen in Polus map (couldn't test on Mira because of the server issues).

Any idea why this might be?

**Comment by edqx** [comment]
Good luck on getting pathfinding to work, I haven't tested that in a long time and I'll be very surprised if it still functions correctly.

But on-topic this looks like an issue with SkeldJS, nothing your fault. I'll look into this shortly for you.

And thanks, although I'm not too sure about the docs being good haha.

**Comment by edqx** [comment]
And thanks for reporting this!

**Comment by Ashesh3** [comment]
> Good luck on getting pathfinding to work, I haven't tested that in a long time and I'll be very surprised if it still functions correctly.
> 
> But on-topic this looks like an issue with SkeldJS, nothing your fault. I'll look into this shortly for you.
> 
> And thanks, although I'm not too sure about the docs being good haha.

Thanks for acknowledging and by the way, pathfinding surprisingly works perfect so far! (only tested for short distances)

**Comment by edqx** [comment]
Fixed in 78dcb4f224dc621978f14718c07a1f238b3129c3! Update to 2.15.3, let me know if you have any more issues :)
